app/views/layouts/application.html.erb

```diff
@@ -11,32 +11,37 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-  <nav class="navbar navbar-default" role="navigation">
+  <nav class="navbar navbar-expand-lg navbar-light bg-light" role="navigation">
     <div class="container">
       <div class="navbar-header">
-        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav-menu">
-          <span class="sr-only">Toggle navigation</span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
         </button>
       </div>
-      <div class="collapse navbar-collapse" id="nav-menu">
+      <div class="collapse navbar-collapse" id="navbarNavDropdown">
```

Comment:
do we have a naming convention that we follow for htmlx?

Reply:
i dont think there is any naming convention

Reply:
we can add some coding standard here. for both ruby and frontend side. this is for the long term health of the project.

usually I recommend Google style guide: https://google.github.io/styleguide/htmlcssguide.html (for JavaScript I think Airbnb has a better standard. Whatever standard it might be, we should setup linter and rules to keep the consistency)

Reply:
I will look into it and see how.